tests/scripts/cute_dsl_kernels/run_custom_op_dense_gemm_swiglu.py

```
@@ -0,0 +1,430 @@
+            continue
+        if k % 16 != 0:
+            print(f"\nSkipping M={m}, N={n}, K={k}: K must be divisible by 16")
+            skipped += 1
+            continue
+
+        try:
+            if test_bf16_output(m, intermediate_size, k):
+                passed += 1
+            else:
+                failed += 1
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)